parser_test.go

```diff
@@ -704,6 +704,7 @@ func (s *testParserSuite) TestDMLStmt(c *C) {
 		{"admin reload bindings", true, "ADMIN RELOAD BINDINGS"},
 		{"admin show telemetry", true, "ADMIN SHOW TELEMETRY"},
 		{"admin reset telemetry_id", true, "ADMIN RESET TELEMETRY_ID"},
+		{"admin reload statistics", true, "ADMIN RELOAD STATISTICS"},
```

Comment:
Is this statement used to update extended statistics?
Why not use a similar syntax to `analyze ...`, for example, `analyze statistics stat1`, `analyze statistics table1`?

Reply:
This is used to reload the stats cache for extended statistics from `mysql.stats_extended`, normally this command is not needed, it is a remedial approach provided for DBA to guarantee the stats cache is consistent with records in `mysql.stats_extended`.

Reply:
OK